lib/gemfury/command/app.rb

```diff
@@ -13,6 +13,7 @@ def version
   end
 
   ### PACKAGE MANAGEMENT ###
+  option :public, :type => :boolean, :desc => "If new package, set privacy to public by default"
```

Comment:
The description seems a bit confusing, particularly "by default" makes me nervous that the customer will think that public is default when the flag is not specified. How about this instead:

```suggestion
  option :public, :type => :boolean, :desc => "Create as public package"
```
or

```suggestion
  option :public, :type => :boolean, :desc => "Creates public package"
```

This describes the primary purpose (setting privacy on creation), and the long manual can talk about other behavior of this flag.